app/main.py

```diff
@@ -2,0 +1,9 @@
+def copy_file(command: str) -> None:
+    if command[0:2] != "cp" or command.count(" ") != 2:
+        return
+    _, file_name, copy_file = command.split(" ")
```

Comment:
Consider renaming the variable `copy_file` to something more descriptive, like `destination_file`, to avoid confusion.